src/SpectatorView.Unity/Assets/SpectatorView/Scripts/SpatialAlignment/SpatialCoordinateSystemManager.cs

```diff
@@ -51,6 +52,55 @@ public class SpatialCoordinateSystemManager : Singleton<SpatialCoordinateSystemM
         private HashSet<INetworkManager> networkManagers = new HashSet<INetworkManager>();
         private ISpatialLocalizationSession currentLocalizationSession = null;
 
+        /// <summary>
+        /// Current Tracking state for the AR/VR Device associated with the application.
+        /// </summary>
+        public TrackingState TrackingState
+        {
+            get
+            {
+                if (!lookedForTrackingObservers)
+                {
+                    List<GameObject> roots = new List<GameObject>();
+                    SceneManager.GetActiveScene().GetRootGameObjects(roots);
+
+                    var observers = new List<ITrackingObserver>();
+                    foreach (var root in roots)
+                    {
+                        var tempObservers = root.GetComponentsInChildren<ITrackingObserver>();
+                        if (tempObservers != null)
+                        {
+                            foreach (var observer in tempObservers)
+                            {
+                                observers.Add(observer);
+                            }
+                        }
+                    }

```

Comment:
If ITrackingObserver became an abstract class TrackingObserver that derives from MonoBehaviour, then all of this logic could be replaced with UnityEngine.Object.FindObjectsOfType<TrackingObserver>();